README.md

```diff
@@ -0,0 +1,67 @@
+# Static Comp
+Module 2: Solo Challenge
+
+## Table of Contents
+  - [Abstract](#abstract)
+  - [Technologies](#technologies)
+  - [Illustrations](#illustrations)
+  - [Install + Setup](#set-up)
+  - [Contributors](#contributors)
+	- [Wins](#wins)
+	- [Challenges + Improvements](#challenges-+-Improvements)
+  - [Project Specs](#project-specs)
+  - [Deployed Site](#depolyed-site)
+
+## Abstract
+  - This project was a challenge to see how well we could build to match a desktop comp as closely as possible, while still having some creative liberties. 
+
+  - My wireframing of the different device sizes can be found [here](https://miro.com/app/board/o9J_l5Onets=/)
+
+## Technologies
+  - Javascript
+  - HTML
+  - CSS (including normalize.css)
+  - node
+  - VSCode
+  - Miro (wireframing)
+
+## Illustrations
+
+  - Here is an image of the comp to match: 
+
+![Screen Shot 2021-07-23 at 2 39 40 PM](https://user-images.githubusercontent.com/78767067/126917173-caaac89d-6bc9-4cb0-b540-7d1e68f714ab.png)
+
+  - I decided to match everything as best as possible to the original as a personal challenge, but did want to add some of my favorite Monet paintings for the image backgrounds. This is how the desktop turned out: 
+
+![Screen Shot 2021-07-25 at 4 27 36 PM](https://user-images.githubusercontent.com/78767067/126917195-6e2da4e4-2858-450a-8613-52e34202c054.png)
+
+  - With this challenge, I jumped into my first attempt at styling "mobile first" to ensure the look would style well on smaller screens.
+
+![Screen Shot 2021-07-25 at 4 28 25 PM](https://user-images.githubusercontent.com/78767067/126917298-abc7440c-77ef-42f3-90a2-b90872ef89e4.png)
+
+  - After the mobile look was set, I utilized the responsive CSS grid format and media queries to ensure the look for iPad and laptop:
+
+![Screen Shot 2021-07-25 at 4 28 08 PM](https://user-images.githubusercontent.com/78767067/126917334-1576544d-2783-4cd7-8703-65105563802d.png)
+
+
+## Install + Setup
+  - Clone this repo to your local computer. 
```

Comment:
I would be explicit about how to do this, too. Include the link from your repo, too. The less work the contributor has to do, the better.